target/sama5d2/board_sama5d2-xplained.h

```diff
@@ -266,7 +266,7 @@
 
 #define QSPIFLASH_PINS     PINS_QSPI0_IOS3
 #define QSPIFLASH_ADDR     QSPI0
-#define QSPIFLASH_BAUDRATE 50000000 /* 50 MHz */
+#define QSPIFLASH_BAUDRATE 100000000 /* 50 MHz */
```

Comment:
Comment is not inline with the newly defined baudrate. More over, it is not inline with the commit message announcement.